### ISSUE TITLE
Everytime I plug in my Q2, the UI just disappears 

### ISSUE THREAD

**Issue by macckerel** [the-via/releases#167]
![Capture](https://user-images.githubusercontent.com/100891559/156704398-606d0e8e-fad2-4b49-8dfb-72ab097134c9.PNG)

**Comment by seagle0128** [comment]
Same issue here, with Q1 knob.

**Comment by suppasit-odds** [comment]
I have the same issue on my Q2 and VIA v1.3.1. To solve the problem, I unplug my Q2, open VIA, import the JSON file to VIA, and then connect Q2 back to my Mac. With these steps, VIA should be able to load successfully.

I suspect that we will have this issue until the new version of VIA or Q2 firmware is released.

**Comment by seagle0128** [comment]
> I have the same issue on my Q2 and VIA v1.3.1. To solve the problem, I unplug my Q2, open VIA, import the JSON file to VIA, and then connect Q2 back to my Mac. With these steps, VIA should be able to load successfully.
> 
> I suspect that we will have this issue until the new version of VIA or Q2 firmware is released.

Unfortunately the solution doesn't work for me 😢

**Comment by Akibahara** [comment]
Same issue here with my Q2 Knob and VIA 1.3.1.   Was able to import JSON file,  but all of a sudden no longer working now.

**Comment by mpkonmbk** [comment]
I have the same issue, @suppasit-odds 's solution works to an extent - I'm able to edit key assignments - but editing lighting makes the UI disappear.

**Comment by lukas4u** [comment]
Yeah. I also have exactly same issue on my Q2. @suppasit-odds ‘s way is work on me. 

Unplug cable > run via > import json > then plug the cable, it works!

**Comment by Fragggy** [comment]
Same Problem for me.
Also if I select Lighting everything disappears again.

**Comment by za59947** [comment]
same for me with Q2

**Comment by SamuelPoulin** [comment]
Same here with q2 non-knob version

**Comment by lukas4u** [comment]
I recommend VIAL instead of VIA.

**Comment by talalashraf** [comment]
anyone here able to find a solution that doesn't involve plugging unplugging the keyboard ?

**Comment by erretter** [comment]
Same problem here. Q2 ISO non Knob. Latest firmware 1.05
